aws/resource_aws_autoscaling_schedule_test.go

```diff
@@ -33,6 +34,47 @@ func TestAccAWSAutoscalingSchedule_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSAutoscalingSchedule_basicImport(t *testing.T) {
+	var schedule autoscaling.ScheduledUpdateGroupAction
+	rName := fmt.Sprintf("tf-test-%d", acctest.RandInt())
+	start := testAccAWSAutoscalingScheduleValidStart(t)
+	end := testAccAWSAutoscalingScheduleValidEnd(t)
+
+	scheduledActionName := "foobar"
+	resourceName := "aws_autoscaling_schedule.foobar"
+	importInput := fmt.Sprintf("%s/%s", rName, scheduledActionName)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSAutoscalingScheduleDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSAutoscalingScheduleConfig(rName, start, end),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckScalingScheduleExists("aws_autoscaling_schedule.foobar", &schedule),
+				),
+			},
+			// Test existent resource import
```

Comment:
Instead of creating a new acceptance test that requires creating duplicate testing infrastructure, the new `ImportStateVerify: true` `TestStep` should be added to all acceptance tests (except `_disappears`) to verify that each resource configuration successfully imports. This can find issues in the resource `Read` function where we are not properly refreshing the Terraform state. The non-existent resource import `TestStep` can be added just to the `_basic` test. 👍

Reply:
Fixed in 644a638